index.html

```diff
@@ -113,17 +113,70 @@ <h2>Multi-Post Stories</h2>
               </li>
             </ul>
           </div>
-          <div class="project-button">
-            <a href="projectlink"
-              ><img
-                src="images/Icons/Button/Enable.png"
-                alt="button
-        icon"
-            /></a>
+          <div class="project-button click-project featureProj">
+            <button type="button" class="see-project liveLink">
+              See Project
+            </button>
           </div>
         </div>
       </div>
     </section>
+
+    <!-- Pop Up Window -->
+    <div class="popup hidden">
+      <!-- title and the cancel button-->
+      <div class="popup-heading">
+        <h2>Multi Post Stories</h2>
+        <img
+          class="popup-close"
+          src="images/popup/close.png"
+          alt="close icon"
+        />
+      </div>
+
+      <!-- popup list consisting of languages-->
+      <ul class="popup-list">
+        <li>html</li>
+        <li>Bootstrap</li>
+        <li>Ruby on Rails</li>
+      </ul>
+
+      <div class="popup-content">
+        <!--Snapshot image of the project-->
+        <img
+          class="image-proj"
+          src="images/popup/desksnap.png"
+          alt="project Snapshot"
+        />
+
+        <!--Paragraph text-->
+
+        <div class="popup-desc">
+          <p class="popup-text">
+            Lorem Ipsum is simply dummy text of the printing and typesetting
+            industry. Lorem Ipsum has been the industry's standard dummy text
+            ever since the 1500s, when an unknown printer took a galley of type
+            and scrambled it to make a type specimen book. It has survived not
+            only five centuries, but also the leap into electronic typesetting,
+            remaining essent
+          </p>
```

Comment:
Please ensure that the projects section is created dynamically with the information from the projects array (all the HTML of the section is created when the page loads). Please remove all the hard-code from your `html file`